operators/db2/service-account/daemon-set.yaml

```diff
@@ -0,0 +1,58 @@
+# Required for the NFS support
+# https://www.ibm.com/docs/en/db2/11.5?topic=SSEPGG_11.5.0/com.ibm.db2.luw.db2u_openshift.doc/aese-cfg-nfs-filegold.html
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: norootsquash
+  namespace: db2
+  labels:
+    tier: management
+    app: norootsquash
+spec:
+  selector:
+    matchLabels:
+      name: norootsquash
+  template:
+    metadata:
+      labels:
+        name: norootsquash
+    spec:
+      serviceAccountName: norootsquash
+      initContainers:
+        - resources:
+            requests:
+              cpu: 0.1
+          securityContext:
+            privileged: true
+          image: alpine:3.6
```

Comment:
Careful!!! See link below:

https://hub.docker.com/layers/alpine/library/alpine/3.6/images/sha256-43773d1dba76c4d537b494a8454558a41729b92aa2ad0feb23521c3e58cd0440?context=explore

Alpine 3.6 is a little old but more importantly the image is not scanned for the log4shell vulnerability. I would advise using a more recent scanned image, as such: 

https://hub.docker.com/layers/alpine/library/alpine/3.15.5/images/sha256-26284c09912acfc5497b462c5da8a2cd14e01b4f3ffa876596f5289dd8eab7f2?context=explore